src/components/RoomView/Card.tsx

```diff
@@ -55,11 +55,10 @@ const Card = (word: CardObj) => {
         updates[`${newHistoryKey}`] = newGameHistory;
         update(gameHistoryRef, updates);
         update(singleCardRef, { isVisibleToAll: true, teamId: cardBelongsTo });
```

Comment:
the below code is no longer dishonest! when an assassin is clicked, the winners and losers are set respectively and the actions to redux in roomview when game is 'complete' are triggered

Reply:
tested & approved